### PR TITLE
[js-md5] Add latest version

### PR DESCRIPTION
**Pull request by martin-badin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test js-md5`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## js-md5 ([<kbd>unpkg</kbd>](https://unpkg.com/browse/js-md5@latest/))
was missing the following properties:
1. blocks
1. buffer8
1. h0
1. h1
1. h2

<details>
<summary>as well as these 11 other properties...</summary>
<p>h3, start, bytes, hBytes, finalized, hashed, first, lastByteIndex, finalize, hash, toString</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c10c28bddd7bdad8244b0642db78e39fa9b6faa1
</p>

**Comment by martin-badin** [comment]
@peterblazejewicz please review. Small PR. Thanks

File: types/js-md5/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for js-md5 0.4
+// Type definitions for js-md5 0.7
```
Comment (peterblazejewicz) [inline types/js-md5/index.d.ts:1]: you're missing 'base64' export, if you intend to bump to .7. Otherwise the change is just niceties.

Comment (martin-badin) [reply types/js-md5/index.d.ts:1]: Thanks. I updated tests https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63603/files#diff-0ea5d1a799ce76d975de29a1a13b3699d83183391934c8e35818397a7f96a916R9

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@martin-badin thanks!

**Comment by martin-badin** [comment]
Ready to merge

**Pull request merged**